distributed/dashboard/tests/test_scheduler_bokeh.py

```diff
@@ -802,6 +804,24 @@ async def test_profile_server(c, s, a, b):
         assert time() < start + 2
 
 
+@pytest.mark.slow
+@gen_cluster(
+    client=True,
+    config={
+        "distributed.worker.profile.enabled": False,
+        "distributed.worker.profile.interval": "10ms",
+        "distributed.worker.profile.cycle": "50ms",
+    },
+)
+async def test_profile_server_disabled(c, s, a, b):
+    ptp = ProfileServer(s)
+    assert "disabled" in ptp.subtitle.text
+    start = time()
+    await asyncio.sleep(1)
```

Comment:
Do you actually need to wait for a second? with the configured intervals/cycles I would expect this to be a fraction of a second such that test runtime is lower

Reply:
I wanted to be on the safe side wrt. CI randomly suspending execution for some time. Writing this down, I realize that this would cause a _false negative_ flake, i.e. this test would not fail even though it is broken, and it should still fail whenever CI does not suspend execution, very much including local machines. So I guess it's okay to reduce the sleep less and have this test occasionally pass on CI should it ever break.

Reply:
Reduced to `100ms`